### PR TITLE
Migrated all statuses to status types

### PR DESCRIPTION
**Pull request by jatin69**
Solves #115 
Replaced all status strings with status types.

### CODE REVIEW

File: content.js [hunk 1]
```diff
@@ -194,18 +194,18 @@ Provider.prototype.checkStatus = function () {
...
 
         case "new.vk.com":
             status = document.querySelector('.top_audio_player')
-                && document.querySelector('.top_audio_player').classList.contains('top_audio_player_playing') ? 'playing' : 'paused';
+                && document.querySelector('.top_audio_player').classList.contains('top_audio_player_playing') ? Status.PLAYING : Status.PAUSED;
             break;
 
         case "last.fm":
-            status = document.getElementById('webRadio').classList.contains('playing') ? 'playing' : 'paused';
+            status = document.getElementById('webRadio').classList.contains(Status.PLAYING) ? Status.PLAYING : Status.PAUSED;
```
[the remark this answers is not in the document]
Comment (jatin69) [reply content.js]: Yes, you are right. I'll update this.

File: content.js [hunk 2]
```diff
@@ -281,82 +281,82 @@ Provider.prototype.checkStatus = function () {
...
             break;
         case "jazzradio.com":
-            status = document.querySelector('#play-button .icon-pause') ? 'playing' : 'paused';
+            status = document.querySelector('#play-button .icon-pause') ? Status.PLAYING : Status.PAUSED;
             break;
         case "v5player.slipstreamradio.com":
             status = document.getElementById('statusLabel') &&
                 document.getElementById('statusLabel')
-                .textContent.toLocaleLowerCase() == 'playing' ? 'playing' : 'paused';
+                .textContent.toLocaleLowerCase() == Status.PLAYING ? Status.PLAYING : Status.PAUSED;
```
Comment (endway) [inline content.js]: `.textContent.toLocaleLowerCase() == 'playing'` this one as well

**Comment by beshur** [comment]
Looks good to me, will approve after testing

File: content.js (hunk 2 above)
Comment (beshur) [reply content.js]: Tested. Other than this place, everything is good.
Please change it and we'll merge it right away.

Thank you, @jatin69!

**Comment by jatin69** [comment]
Hi @beshur  I made all the required changes & squashed commits. 
Thank you for being so patient with me. 🔥

**Pull request merged**